features/cli.feature

```diff
@@ -1,5 +1,18 @@
 Feature: `wp cli` tasks
 
+  Scenario: Ability to detect a WP-CLI registered command
+    Given an empty directory
+    And save the {SRC_DIR}/VERSION file as {TRUE_VERSION}
+    And a new Phar with version "1.3.0"
+
+    When I try `{PHAR_PATH} cli has-command scaffold package`
+    Then the return code should be 1
+
+    When I run `{PHAR_PATH} package install git@github.com:wp-cli/scaffold-package-command.git`
```

Comment:
Use a short package identifier here:
`When I run `{PHAR_PATH} package install wp-cli/scaffold-package-command`

Reply:
@schlessera, Why didn't git clone work?

Reply:
Probably just because Travis hit the git rate limiting.

With the short package identifier, in this case, you fetch the ZIP through the old package index, instead of the GH versioned source.

Reply:
I see :eyes: